misc-tools/dj_judgehost_cleanup.in

```diff
@@ -17,7 +17,9 @@ Perform various cleanup tasks on a judgehost.
 
 Make sure that no judgings are in process while running this script.
 It is best to temporarily disable the judgehost in the web interface
-or stop the service.
+or stop the service and wait until all judgings are finished.
```

Comment:
Nothing will finish if you stop the service? So don't really understand this addition?